src/rluni/motors/motors.py

```diff
@@ -159,6 +159,12 @@ async def set_torque(self, torque=0.0, max_torque=0.2, query=False):
         Raises:
             asyncio.TimeoutError: If the operation times out.
         """
+        # fail if we are a generic motor 
```

Comment:
Have a look at the controllerABC. This will work fine but if someone changes the default model name it will break this function. 

Is the intention here to have a generic motor available that will refuse torque commands? We could have a private field torque_permitted that is set to False, which the MN motors override to true. The motor model field is duplicate of the class name (which can be fetched from a any variable with __class__.__name__ ). We can leave that there, but I'd prefer not to have the model name field control the torque conditional.